inference-engine/src/mkldnn_plugin/nodes/mkldnn_matmul_node.cpp

```diff
@@ -90,14 +89,18 @@ void MKLDNNMatMulNode::setPostOps(mkldnn::primitive_attr &attr, bool initWeights
 }
 
 
-MKLDNNNode::AttrPtr MKLDNNMatMulNode::initPrimitiveAttr() const {
+MKLDNNNode::AttrPtr MKLDNNMatMulNode::initPrimitiveAttr(const VectorDims &dims) const {
     auto attr = std::make_shared<mkldnn::primitive_attr>(mkldnn::primitive_attr());
 
-    setPostOps(*attr, true);
+    setPostOps(*attr, dims, true);
 
     return attr;
 }
 
+MKLDNNNode::AttrPtr MKLDNNMatMulNode::initPrimitiveAttr() const {
+    return initPrimitiveAttr(MemoryDescUtils::makeDummyShape(getInputShapeAtPort(0)).getStaticDims());
```

Comment:
as far as I remember `initPrimitiveAttr` used inside `initDescriptor`
so for dynamic case you got exception
so maybe better use `getDims` ?

Reply:
Please note that here we call `MemoryDescUtils::makeDummyShape` and only then we call `getStaticDims()`, so the dummy shape is always static by its definition. But, thanks for the comment anyway. Since the code is not obvious enough I decided to split this oneliner to a few lines.